flocker/control/httpapi.py

```diff
@@ -568,14 +606,24 @@ def container_configuration_response(application, node):
     """
     result = {
         "host": node, "name": application.name,
-        "image": application.image.full_name
+        "image": application.image.full_name,
+        "restart_policy": {
+            "name": application.restart_policy.name
```

Comment:
See comment above; I think we should dictionary for this, not rely on attribute of `IRestartPolicy`.